packages/vitest/src/create/browser/examples.ts

```diff
@@ -135,6 +135,62 @@ test('renders name', async () => {
 `,
 }
 
+const litExample = {
+  name: 'HelloWorld.js',
+  js: `
+import { html, LitElement } from 'lit'
+
+export class HelloWorld extends LitElement {
+  static properties = {
+    name: { type: String },
+  }
+
+  constructor() {
+    super()
+    this.name = 'World'
+  }
+
+  render() {
+    return html\`<h1>Hello \${this.name}!</h1>\`
+  }
+}
+
+customElements.define('hello-world', HelloWorld)
+`,
+  ts: `
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+@customElement('hello-world')
+export class HelloWorld extends LitElement {
+  @property({ type: String })
+  name = 'World'
+
+  render() {
+    return html\`<h1>Hello \${this.name}!</h1>\`
+  }
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'hello-world': HelloWorld
+  }
+}
+`,
+  test: `
+import { expect, test } from 'vitest'
+import { render } from 'vitest-browser-lit'
+import { html } from 'lit'
+import './HelloWorld.js'
+
+test('renders name', async () => {
+  const screen = render(html\`<hello-world name="Vitest"></hello-world>\`)
+  const element = screen.getByText('Hello Vitest!')
+  expect(element).toBeInTheDocument()
```

Comment:
I was testing through `pnpm dlx https://pkg.pr.new/vitest@7705 init browser` and spotted this one is failing:

```
 FAIL   chromium  vitest-example/HelloWorld.test.ts > renders name
Error: expect(received).toBeInTheDocument()

received value must be an HTMLElement or an SVGElement.

 ❯ vitest-example/HelloWorld.test.ts:10:18
      8|   const element = screen.getByText('Hello Vitest!')
      9|   // await expect.element(element).toBeInTheDocument()
     10|   expect(element).toBeInTheDocument()
       |                  ^
     11| })
     12| 
```

```suggestion
  await expect.element(element).toBeInTheDocument()
```

I see marko and vanilla has `expect(element).toBeInTheDocument()`, but they might be different since they use testing-library?

Reply:
yep, that's a good spot - element here will be a locator, so we should use expect.element for retryability :)